assets/js/helpers/contains.js

```diff
@@ -9,21 +9,12 @@
 function contains(line, list) {
     var lines = list.split('\n');
 
-    for (var i = 0; i < lines.length; i ++) {
-
-        // Trim all lines from the list one by one
-        var cleanLine = lines[i].trim();
-
-        // If by any chance one line in the list is empty, ignore it
-        if(cleanLine === '') continue;
-
-        // If line contains the clean line return true
-        if (line.indexOf(cleanLine) > - 1) {
-            return true;
-        }
+    for(var i in lines) {
+        // Use regex to remove all whitespace
+        var cleanLine = lines[i].replace(/\s/g, '');
```

Comment:
This is cool. I always forget the regex syntax but I believe that this will be good.